### PR TITLE
Make supported-ciphersuites be mandatory in QueryRequest

### LINKED ISSUE #222: Hackathon114: Make supported-ciphersuites be mandatory in QueryRequest
Currently supported-ciphersuites is optional which means we must specify what is meant when it is absent.
There are many possibilities, and all 5 implementers at the hackathon agreed that making it be mandatory is probably simpler. 
Ken also pointed out that while there are 2 ciphersuites defined as mandatory now, those could change in the future and making the TAM put them into the QueryRequest message explicitly is more future proof.

### PR DESCRIPTION
**Pull request by dthaler**
Addresses #222

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

### CODE REVIEW

**Comment by dthaler** [comment]
@kentakayama @ko-isobe please review

File: draft-ietf-teep-protocol.cddl [hunk 1]
```diff
@@ -36,9 +36,9 @@ $data-item-requested /= extensions
 
 query-request = [
   type: TEEP-TYPE-query-request,
+  supported-ciphersuites: [ + ciphersuite ],
```
Comment (ko-isobe) [inline draft-ietf-teep-protocol.cddl:39]: move this line between L47 and L48 to align with draft markdown.

File: draft-ietf-teep-protocol.md [hunk 2]
```diff
@@ -1585,25 +1584,29 @@ This section includes some examples with the following assumptions:
 {: numbered='no'}
 
 ~~~~
-83                  # array(3)
+84                  # array(4)
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:1587]:
```suggestion
85                  # array(5)
```

**Review by mcd500 — APPROVED** [review]
Merging

**Pull request merged**